docs/getting-started/install/linux/index.md

```diff
@@ -87,6 +90,8 @@ The package ***referenceassemblies-pcl*** should be installed for PCL compilatio
 
 The package ***ca-certificates-mono*** should be installed to get SSL certificates for HTTPS connections. Install this package if you run into trouble making HTTPS connections.
 
+The package ***msbuild*** may be needed to compile very new projects which are not compatible with the deprecated MSBuild replacement, XBuild.
+
 The module ***mono-xsp4*** should be installed for running ASP.NET applications.
 
 Notes
```

Comment:
It's still accurate for the named releases. We could ditch the section if we're disclaiming responsibility for the versions listed, but would need to bring it back in future if any new vulns are found & we offer security updates to old versions.